controllers/ReviewController.ts

```diff
@@ -0,0 +1,57 @@
+import {Request, Response, Express} from "express";
+import ReviewDao from "../daos/ReviewDao";
+import ReviewControllerI from "../interfaces/ReviewControllerI";
+
+export default class ReviewController implements ReviewControllerI {
+    private static reviewController: ReviewController | null = null;
+    private static reviewDao: ReviewDao = ReviewDao.getInstance();
+
+    public static getInstance = (app: Express): ReviewController => {
+        if (ReviewController.reviewController === null) {
+            ReviewController.reviewController = new ReviewController();
+            app.get("/api/reviews", ReviewController.reviewController.findAllReviews);
+            app.get("/api/restaurant/:restaurantid/reviews", ReviewController.reviewController.findAllReviewsForRestaurant);
```

Comment:
Hi Am, could you please change restaurant to plural "restaurants" for all the HTTP endpoints?